app/config/Config.scala

```diff
@@ -40,6 +45,19 @@ class Config(playConfig: Configuration) extends Logging {
 
   lazy val localLogShipping: Boolean = sys.env.getOrElse("LOCAL_LOG_SHIPPING", "false").toBoolean
   lazy val loggingStreamName: Option[String] = playConfig.getOptional[String]("recipes.loggingStreamName")
+  
+  lazy val identity = stage match {
+    case Dev => new DevIdentity(s"$app")
+    case _ => new AwsIdentity(s"$app", s"$stack", s"$stage", s"${awsRegion}")
```

Comment:
We can use the variables directly here as they are of the correct type. That is, we don't need to coerce them into strings.

```suggestion
    case Dev => new DevIdentity(app)
    case _ => new AwsIdentity(app, stack, stage, awsRegion)
```